examples/radio/radio-rating.html

```diff
@@ -0,0 +1,290 @@
+              <td>
+                <ul>
+                  <li>Moves focus to and checks the next <code>radio</code> button in the group.</li>
+                  <li>If focus is on the last <code>radio</code> button, moves focus to the first <code>radio</code> button.</li>
+                  <li>The state of the previously checked radio button is changed to unchecked.</li>
+                </ul>
+              </td>
+            </tr>
+            <tr data-test-id="key-up-left-arrow">
+              <th><kbd>Up arrow</kbd><br/><kbd>Left arrow</kbd></th>
+              <td>
+                <ul>
+                  <li>Moves focus to and checks the previous <code>radio</code> button in the group.</li>
+                  <li>If focus is on the first <code>radio</code> button, moves focus to and checks the last <code>radio</code> button.</li>
+                  <li>The state of the previously checked radio button is changed to unchecked.</li>
+                </ul>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </section>
+
+<section>
+        <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
+        <table aria-labelledby="rps_label" class="data attributes">
+          <thead>
+            <tr>
+              <th scope="col">Role</th>
+              <th scope="col">Attributes</th>
+              <th scope="col">Element</th>
+              <th scope="col">Usage</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr data-test-id="svg-none">
+              <th scope="row">
+                <code>none</code>
+              </th>
+              <td></td>
+              <td>
+                <code>svg</code>
+              </td>
+              <td>
+                The use of the <code>none</code> role on the SVG element ensures assistive technologies do not interpret the SVG element as an image or some other role.
+              </td>
+            </tr>
+            <tr data-test-id="radiogroup-role">
+              <th scope="row"><code>radiogroup</code></th>
+              <td></td>
+              <td><code>div</code></td>
+              <td>
+                <ul>
+                  <li>Identifies the <code>div</code> element as a container for a group of <code>radio</code> buttons.</li>
+                  <li>Is not focusable because focus is managed using a roving tabindex strategy as described below.</li>
+                </ul>
+              </td>
+            </tr>
+            <tr data-test-id="radiogroup-aria-labelledby">
+              <td></td>
+              <th scope="row"><code>aria-labelledby="[IDREF]"</code></th>
+              <td><code>div</code></td>
+              <td>Refers to the element that contains the label of the radio group.</td>
+            </tr>
+            <tr data-test-id="radio-role">
+              <th scope="row"><code>radio</code></th>
+              <td></td>
+              <td><code>div</code></td>
```

Comment:
Looking at the code, I think this should be a `<g>` element rather than a `<div>` element. Is that correct?